### PR TITLE
logout button & fixes

### PR DESCRIPTION
**Pull request by siddhigate**
- Logout button in header navigation bar
- Nav links fix in pages

**Preview**: https://fashionly-ui-git-authentication-pages-siddhigate.vercel.app/pages/authentication/logoutheader.html

image

### CODE REVIEW

**Review by ZuberDunge — APPROVED** [review]
Well structured code, clean and readable. Logout page looks awesome, Good work Siddhi. Good to go from my side.

**Review by Manju-sree — APPROVED** [review]
better to have this in utilities, rest code looks good to merge

File: css/pages/authentication.css [hunk 1]
```diff
@@ -24,6 +24,17 @@
   font-weight: var(--fw-bold);
 }
 
+.auth-form .input-password-wrapper {
+  position: relative;
+}
+
+.input-password-wrapper i {
+  position: absolute;
```
Comment (Manju-sree) [inline css/pages/authentication.css:28]: better to have this in utilities

**Review by Manju-sree — APPROVED** [review]
added a small suggestion apart from that code looks good to merge

File: css/pages/authentication.css (hunk 1 above)
Comment (siddhigate) [reply css/pages/authentication.css:28]: Thanks for your suggestion! Have made the changes accordingly

**Pull request merged**